hash.go

```diff
@@ -314,3 +319,53 @@ func (db *RoseDB) HScan(key []byte, prefix []byte, pattern string, count int) ([
 	}
 	return values, nil
 }
+
+// HIncrby increments the number stored at field in the hash stored at key by increment.
+// If key does not exist, a new key holding a hash is created.
+// If field does not exist the value is set to 0 before the operation is performed.
+// The range of values supported by HINCRBY is limited to 64bit signed integers.
```

Comment:
You can move a partial upper line at the beginning of this line to use the space efficiently.

```suggestion
// HIncrby increments the number stored at field in the hash stored at key by increment.
// If key does not exist, a new key holding a hash is created. If field does not exist
// the value is set to 0 before the operation is performed. The range of values supported
//  by HINCRBY is limited to 64bit signed integers.
```

Reply:
@gozeloglu 
Thank you for your suggestion.
It's ok!